test/language/module-code/lex-and-var.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) Copyright 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 10.2.1
+description: >
+    It is a Syntax Error if any element of the LexicallyDeclaredNames of
+    ModuleItemList also occurs in the VarDeclaredNames of ModuleItemList.
+flags: [module]
+features: [let]
```

Comment:
Did you intentionally omit `negative: SyntaxError` here, because the test runners don't yet understand the `module` flag?